tests/e2e/default-mobile/wdio.conf.js

```diff
@@ -6,7 +6,9 @@ chai.use(require('chai-exclude'));
 // Override specific properties from wdio base config
 exports.config = Object.assign(wdioBaseConfig.config, {
   specs: [
-    '**/*.wdio-spec.js'
+    '**/*.wdio-spec.js',
+    '../default/navigation/navigation.wdio-spec.js',
+    '../default/navigation/hamburger-menu.wdio-spec.js',
```

Comment:
I didn't find a way to extend those tests similarly to how we do with desktop and mobile page objects... I would need to research more.

For now, this is okay; we need to keep an eye so people don't start mixing particular logic in desktop tests. 
If you prefer to group them by concerns, wdio has this thing of wrapping [them in brackets](https://webdriver.io/docs/organizingsuites/#grouping-test-specs-to-run-sequentially).